### PR TITLE
Remove dist folder before new build

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
   },
   "scripts": {
     "prebuild": "run-s test",
-    "build": "microbundle-crl --no-compress --format modern,cjs --css-modules false",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
+    "build": "rm -rf dist && microbundle-crl --no-compress --format modern,cjs --css-modules false",
+    "start": "rm -rf dist && microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
```
Comment (jkilzi) [inline package.json:17]: This is unrelated to your PR but if we are already discussing this stuff, then...

```suggestion
    "start": "rm -rf dist && microbundle-crl watch --no-compress --format modern --css-modules false",
```

In watch mode do we really need to create a cjs bundle?

**Review by jkilzi — COMMENTED** [review]
FYI

File: package.json (hunk 1 above)
Comment (rawagner) [reply package.json]: +1
cjs removed

**Comment by rawagner** [comment]
> Why do we need this?

generally we should clean dist folder before building to make sure we do not have some leftovers for previous builds there

> Isn't sync-dist.sh taking care of that?

it does not, if Im looking correclty. Also you may not be using `sync-dist.sh`, just running `yarn build`

**Comment by rawagner** [comment]
@jkilzi updated

**Review by jkilzi — APPROVED** [review]
LGTM

**Pull request merged**